airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/gradle.py

```diff
@@ -102,7 +111,7 @@ async def _run(self, *args: Any, **kwargs: Any) -> StepResult:
             # Set GRADLE_HOME to the directory which will be rsync-ed with the gradle cache volume.
             .with_env_variable("GRADLE_HOME", self.GRADLE_HOME_PATH)
             # Same for GRADLE_USER_HOME.
-            .with_env_variable("GRADLE_USER_HOME", self.GRADLE_HOME_PATH)
+            .with_env_variable("GRADLE_USER_HOME", self.GRADLE_HOME_PATH).with_env_variable("AIRBYTE_LOG_SUBDIR", self.airbyte_logs_subdir)
```

Comment:
can you put this call on a new line with a comment explaining the purpose of the env var please?